source/chapters/user_interface.rst

```diff
@@ -103,27 +123,27 @@ By default the waveform displays are aligned parallel to each other in
 order to make :term:`beatmatching` easier, as it is possible to beatmatch
 visually by aligning the beats that appear in each waveform.
 
-.. figure:: ../_static/Mixxx-200-Latenight-parallel-waveform.png
+.. figure:: ../_static/2.2/deere-parallel-waveforms.png
    :align: center
    :width: 100%
    :figwidth: 100%
-   :alt: Mixxx default skin (Latenight) - Parallel waveforms
+   :alt: Mixxx default skin (Deere) - Parallel waveforms
    :figclass: pretty-figures
 
-   Mixxx default skin (Latenight) - Parallel waveforms
+   Mixxx default skin (Deere) - Parallel waveforms
 
 Depending on the skin, Mixxx displays either parallel waveforms (default) or
 separate waveforms. Select your preferred appearance in
 :menuselection:`Preferences --> Interface --> Skin`.
 
-.. figure:: ../_static/Mixxx-200-Deere-separate-waveform.png
+.. figure:: ../_static/2.2/latenight-parallel-waveforms.png 
```

Comment:
There already is a screenshot of Deere with parallel waveforms. Instead of a LateNight screenshot with parallel waveforms, this should be a screenshot of Deere with separate waveforms.

Reply:
Okay. Let me change this.

Reply:
I have made the requested changes @Holzhaus . Ready for your review... :)